rendermodules/em_montage_qc/detect_montage_defects.py

```diff
@@ -0,0 +1,270 @@
+from functools import partial
+from rtree import index as rindex
+from scipy.spatial import KDTree
+import matplotlib.pyplot as plt
+import networkx as nx
```

Comment:
Can we modify requirements.txt with the packages these modules require?  I am seeing networkx, rtree, matplotlib (which appears to be unused), bokeh, and shapely which are not requirements yet.

Reply:
networkx, rtree and bokeh are being used. matplotlib was used by some commented out code (removed them now). Shapely is not being used by any of these code, so did not add it to requirements.